architecture/new_schema.sql

```diff
@@ -0,0 +1,1060 @@
+    ADD CONSTRAINT "PK_1f64a9d131c0f7245a90deee93f" PRIMARY KEY ("messageId", "credentialHash");
+
+
+--
+-- Name: presentation_credentials_credential PK_32d9cee791ee1139f29fd94b5c4; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.presentation_credentials_credential
+    ADD CONSTRAINT "PK_32d9cee791ee1139f29fd94b5c4" PRIMARY KEY ("presentationHash", "credentialHash");
+
+
+--
+-- Name: credential PK_32ea339ef30d340caac7961bd44; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.credential
+    ADD CONSTRAINT "PK_32ea339ef30d340caac7961bd44" PRIMARY KEY (hash);
+
+
+--
+-- Name: service PK_85a21558c006647cd76fdce044b; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.service
+    ADD CONSTRAINT "PK_85a21558c006647cd76fdce044b" PRIMARY KEY (id);
+
+
+--
+-- Name: presentation PK_8731bc1d6170def561c4820364b; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.presentation
+    ADD CONSTRAINT "PK_8731bc1d6170def561c4820364b" PRIMARY KEY (hash);
+
+
+--
+-- Name: private-key PK_87ac66202b01336a9df721d4ed8; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public."private-key"
+    ADD CONSTRAINT "PK_87ac66202b01336a9df721d4ed8" PRIMARY KEY (alias);
+
+
+--
+-- Name: migrations PK_8c82d7f526340ab734260ea46be; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.migrations
+    ADD CONSTRAINT "PK_8c82d7f526340ab734260ea46be" PRIMARY KEY (id);
+
+
+--
+-- Name: key PK_9bda40833be3c080825245b1a11; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.key
+    ADD CONSTRAINT "PK_9bda40833be3c080825245b1a11" PRIMARY KEY (kid);
+
+
+--
+-- Name: message_presentations_presentation PK_9dc4cc025ec7163ec5ca919d140; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.message_presentations_presentation
+    ADD CONSTRAINT "PK_9dc4cc025ec7163ec5ca919d140" PRIMARY KEY ("messageId", "presentationHash");
+
+
+--
+-- Name: identifier PK_a1ec7926e246e12a63e377e592e; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.identifier
+    ADD CONSTRAINT "PK_a1ec7926e246e12a63e377e592e" PRIMARY KEY (did);
+
+
+--
+-- Name: message PK_ba01f0a3e0123651915008bc578; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.message
+    ADD CONSTRAINT "PK_ba01f0a3e0123651915008bc578" PRIMARY KEY (id);
+
+
+--
+-- Name: presentation_verifier_identifier PK_c3b760612b992bc75511d74f6a9; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.presentation_verifier_identifier
+    ADD CONSTRAINT "PK_c3b760612b992bc75511d74f6a9" PRIMARY KEY ("presentationHash", "identifierDid");
+
+
+--
+-- Name: claim PK_d8d95bd745ab41510c724a43c36; Type: CONSTRAINT; Schema: public; Owner: veramo
+--
+
+ALTER TABLE ONLY public.claim
+    ADD CONSTRAINT "PK_d8d95bd745ab41510c724a43c36" PRIMARY KEY (hash);
+
```

Comment:
If I understand, these are references to a specific table and not relevant outside of it.